packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/template.ts

```diff
@@ -0,0 +1,15 @@
+import { html, when } from '@microsoft/fast-element';
+import type { RichTextMentionUsersView } from '.';
+import { iconAtTag } from '../../../../icons/at';
+
+export const template = html<RichTextMentionUsersView>`<template>
+    ${when(
+        x => x.disableEditing,
+        html<RichTextMentionUsersView>`
+    <span
+        class="control"
+        part="control"
+><${iconAtTag}></${iconAtTag}>${x => x.mentionLabel}</span>
+`
+    )}<slot contenteditable="true"></slot
```

Comment:
Is setting contenteditable required? It doesn't seem to do anything in storybook, i.e. I can't edit the text in the slot.

Reply:
Yes, it is required in editing mode as the slot element is within a shadow root, the parent contenteditable is not inherited when `@` is entered into the editor.

Reply:
Can you describe the expected behavior in storybook? On the editing enabled story: https://60e89457a987cf003efc0a5b-qxfzssfcgg.chromatic.com/?path=/story/tests-rich-text-mention-user--rich-text-mention-user-view-edit-enabled-theme-matrix

I can't type anything into the slot / edit the john doe text. The html looks like:
![image](https://github.com/ni/nimble/assets/1588923/686f9809-0921-492f-8ca7-03c4f5817bed)

If I change the html in devtools to the following:
![image](https://github.com/ni/nimble/assets/1588923/c2cb2004-06b8-4712-8408-8452afad46e2)

Then I can edit the content in the page. The element becomes editable

Reply:
I have to update the storybook on this test so that we can check how it works in edit mode. I actually meant it is required in the editor which is a contenteditable div.

Reply:
I added a contenteditable div to the matrix tests around the element to try editing the component.

Reply:
Since we couldn't find a better approach to solve the problem, we will have the same solution which is to have a contenteditable in slot element. It would be really helpful if you could guide us in a way to find a solution for this issue. Otherwise, the only option is to keep the element editable that functions well in all browsers as it is now. Please let me know your thoughts.

Reply:
The root cause of the issue is not identified yet so we can't strongly say that this issue is related to the Chrome browser or is due to the Tiptap editor handling the decorative node. So I have created a tech debt issue as you mentioned and added it as a comment in the template. Link to the issue: https://github.com/ni/nimble/issues/1659